main.go

```diff
@@ -51,6 +52,7 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 	utilruntime.Must(appstudiov1alpha1.AddToScheme(scheme))
 	utilruntime.Must(appstudioshared.AddToScheme(scheme))
+	utilruntime.Must(gitopsbackendv1alpha1.AddToScheme(scheme))
```

Comment:
what is this for?

Reply:
To get access to GitOpsDeployment. I was testing the full integration which turned out to be not possible. I guess I forgot to remove that.

Reply:
When I look at the latest files changed, this is still present

Reply:
Solved. The rebase put it back.